cli/cmd/plugin/ui/web/tanzu-ui/src/shared/components/management-cluster-settings-step/ManagementClusterSettings.tsx

```diff
@@ -63,7 +63,7 @@ function ManagementClusterSettings(props: Partial<MCSettings>) {
         setSelectedProfile(event.target.value);
     };
 
-    const handleClusterNameChange = (event: ChangeEvent<HTMLInputElement>) => {
+    const handleClusterNameChange = (event: any) => {
```

Comment:
I'm curious why we would abandon the type specification here?

Reply:
good catch. I was trying something else, but didn't change it back.